san-francisco-tourism/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Wed Nov 21 14:27:12 GMT 2018
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
your Gradle wrapper versions do not match, can you use the same version in both:
- san-francisco-tourism-solution/gradle/wrapper/gradle-wrapper.properties
- san-francisco-tourism/gradle/wrapper/gradle-wrapper.properties